docs/metrics.md

```diff
@@ -124,39 +109,44 @@ Services.telemetry.scalarSet(
 	"category.scalar_name", value
 );
 ```
-e.g. `lockboxV1.datastoreCount` for the scalar name.
+e.g. `lockbox-addon.loginCount` for the scalar name.
 
 We can also use `scalarAdd` to increment a scalar value by some amount.
 
 ## Scalar Metrics
 
-These are the metrics we currently collect regarding the state of user datastores.
+These are the metrics we currently collect regarding the state of the user's login storage.
+
+- `loginCount` (SCALAR_TYPE_COUNT). Current count of the number of items in the user's login storage. If possible, this scalar should be set upon opening either the doorhanger or full extension interface.
+
+- `loginsAccessedAllTime` (SCALAR_TYPE_COUNT). Running total of logins accessed (copied, revealed) through the extension interface or doorhanger. Should never be reset.
+
+- `loginsAccessed` (SCALAR_TYPE_COUNT). Total of logins accessed (copied, revealed) since last ping through the extension interface or doorhanger. TODO: in native code scalars are automatically reset when a new ping is cut. Will that happen for us?
+
+Note that if the preceding two scalars are going to cause engineering headaches its possible to obtain equivalent values from the event telemetry (its just more work).
 
-- `datastoreCount` (integer). Current count of the number of items in the user's datastore. Note that this scalar is only updated when the user renders their full item list, either in the management view or in the doorhanger. So when testing whether this scalar is accurately updated, please re-render the item list.
+## List of Planned Metrics Events
 
-## List of Events Currently Recorded
+All events are currently implemented under the **category: lockbox-addon**. The `extra` field contains `itemid` for events pertaining to a particular Lockbox item. They are listed and grouped together below based on the contents of the event's `method` field.
 
-All events are currently implemented under the **category: lockboxV2**. The `extra` field contains `itemid` for events pertaining to a particular Lockbox item. They are listed and grouped together below based on the contents of the event's `method` field.
+1. `startup` fires when the webextension is loaded. **objects**: webextension. Note that this event fires whenever the browser is started, so is not indicative of direct user interaction with Lockbox. **value** is null.
 
-1. `startup` fires when the webextension is loaded. **objects**: webextension. Note that this event fires whenever the browser is started, so is not indicative of direct user interaction with Lockbox.
+2. `click` fires when someone clicks on one of the UI elements listed **objects**: `toolbar`, `get_mobile`, `faq`, `account_settings`, `give_feedback`, `settings_menu`, `reconnect_sync`, `signin_sync` (or whatever the menu is that contains links to account settings, faq, etc). **value** is null.
```

Comment:
Do we want a separate event if users do things with the keyboard instead of the mouse? (Accessibility and keyboard-centric power users)

Reply:
Good question. I don't see myself utilizing that information, but if it would help engineering to have that data I'm happy to add it. And I should say more broadly, if there is any data that engineering would find useful to collect, we should add that.

Reply:
I guess it's really a question of whether product would find the key vs. click (vs. tap, since some desktops have touch screens...) distinction useful or not.

Reply:
@mozilla-lockbox/product is the distinction (click vs tap vs keypress) useful to track?

Reply:
It'd be cool to track but I don't think it's going to give us relevant insights on their own. For now, we probably want to just track that someone did the thing. We can always layer in the distinction at a later time when we have a specific need to identify the methods by which people navigate.